person.rb

```diff
@@ -13,6 +13,10 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
```

Comment:
- [ ] As per [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md), kindly make sure to create a class `Nameable` with a method `correct_name` that will raise a `NotImplementedError` ✨
- [ ] Also, please make sure that your `Person` class inherits from `Nameable` ⛏